docs/local_development.md

```diff
@@ -61,6 +61,10 @@ cd ./aggregator
 cp .env.local.example .env
 ```
 
+> **Note: The .env file in the aggregator needs to be updated everytime the network changes from local to remote and vice-versa.**
+
+> **.env.local.example** for **local**
```

Comment:
> **.env.local.example** for **local**

I'd say you could remove this line, as it is stated in the code example above. Feel free to disagree!